_docs/wallet/ledger-nano-s.md

```diff
@@ -129,7 +129,7 @@ Using the [Ledger Live](https://www.ledger.com/pages/ledger-live) application, f
 	- An installation window will appear, and your device will display **Processing...**
 	- The QRL App installation has completed on your Ledger Nano S
 
-## Initialising the QRL App
+## Initializing the QRL App
```

Comment:
We should probably stick to "s" or "z" both in this heading and the text below... with this edit there is inconsistency with the body text under this heading.

Reply:
Sure, per the answer here https://english.stackexchange.com/a/708 there is no wrong usage, as long as its consistent throughout. As it seems more spell check systems are agreeable to the American version, I will proceed with the usage as -zation throughout.